plugins/org.jboss.tools.locus.mockito/1.9.5/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,101 @@
+eclipse.preferences.version=1
```

Comment:
We could add these ignores to the plugin's .settings... or you could just set these locally. Not sure what the preferred convention is re: committing .settings files.